app/templates/src/test.js

```diff
@@ -1,7 +1,7 @@
-import F from 'funcunit';
-import QUnit from 'steal-qunit';
+var F = require('funcunit');
+var QUnit = require('steal-qunit');
 
-import '<%= name %>/models/test';
+require('<%= name %>/models/test');
```

Comment:
Breaks without it, the FuncUnit patch fixes this. Just making sure the build actually passes now.